jxp/My-List-Util/t/sum.t

```diff
@@ -0,0 +1,18 @@
+#!perl -T
+use 5.006;
+use strict;
+use warnings;
+use Test::More tests => 1;
```

Comment:
Might it have to do with your summing invalid values test printing to `STDOUT`? I had something similar in one of my `Animal` tests - the printed output was treated as a _not ok_ result. If you wrap the whole operation in the `is` I _think_ that discards writes to stdout.